resources/views/livewire/ticket/ticket.blade.php

```diff
@@ -76,25 +95,26 @@
                         </div>
                     </x-slot:header>
                     <div class="space-y-4">
-                        <x-state wire:model.live="ticketState" formatters="formatter.state" avialable="availableStates"/>
+                        <x-state wire:model.live="ticket.state" formatters="formatter.state" available="availableStates"/>
```

Comment:
as this change will always be active after pressing save it doesnt have to be live anymore i guess